lib/app_manifest/manifest.rb

```diff
@@ -13,6 +15,7 @@ def self.from_json(string)
     end
 
     def initialize(hash)
+      AppManifest::ManifestValidator.validate!(hash)
```

Comment:
maybe we can add `validate: true` so clients can `validate: false` if they have a reason to do so